spring-kafka/src/test/java/org/springframework/kafka/listener/AbstractConsumerSeekAwareTests.java

```diff
@@ -88,8 +88,8 @@ void seekForSpecificGroup() throws Exception {
 		MultiGroupListener.latch2 = new CountDownLatch(2);
 
 		multiGroupListener.seekToBeginningForGroup("group2");
-		assertThat(MultiGroupListener.latch1.getCount()).isEqualTo(2);
 		assertThat(MultiGroupListener.latch2.await(10, TimeUnit.SECONDS)).isTrue();
+		assertThat(MultiGroupListener.latch1.await(100, TimeUnit.MICROSECONDS)).isFalse();
```

Comment:
The previous assert validation was incorrect because it had no wait time, so I fixed it.